test/end-to-end/edit_journal/edit_journal.spec.js

```diff
@@ -0,0 +1,146 @@
+/* global browser, element, by */
+const chai = require('chai');
+const helpers = require('../shared/helpers');
+const FU = require('../shared/FormUtils');
+const GU = require('../shared/GridUtils');
+const components = require('../shared/components');
+
+helpers.configure(chai);
+const expect = chai.expect;
+
+describe('Edit Posting Journal', () => {
+  const path = '#!/journal';
+  const gridId = 'journal-grid';
+  before(() => helpers.navigate(path));
+
+  var accountKey = 11;
+
+  it('edits a transaction change an account', function () {
+    // click the "grouping" button
+    FU.buttons.grouping();
+    element.all(by.css('[class="ui-grid-icon-plus-squared"]')).get(0).click();
+    element.all(by.css('[class="fa fa-edit"]')).get(0).click();
+    const accountNumberCell = GU.getCellName(gridId, 1, 4);
+    // simulate dbl click
+    browser.actions().mouseMove(accountNumberCell).doubleClick().perform();
+    
+    accountNumberCell.element(by.css("input")).sendKeys(protractor.Key.chord(protractor.Key.CONTROL, "a"));
```

Comment:
Wow, this is really cool.  I didn't know you could do that.

Protractor has a method for clearing inputs ... does this not work?  It should be:
```js
accountNumberCell.element(by.css("input")).clear();
```
[Documentation](http://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.clear)